core/arch/arm/kernel/abort.c

```diff
@@ -355,11 +456,25 @@ void abort_handler(uint32_t abort_type, struct thread_abort_regs *regs)
 	case FAULT_TYPE_IGNORE:
 		break;
 	case FAULT_TYPE_USER_TA_PANIC:
+#ifdef CFG_WITH_VFP
+		vfp_disable();
+#endif
```

Comment:
Could this be made conditional in the .h instead? Same below for `thread_kernel_save_vfp()` and `thread_kernel_restore_vfp()`.